v3p/zlib/zconf.h

```diff
@@ -151,7 +145,7 @@
 #    define FAR far
 #  endif
 #endif
-#if defined(__BORLANDC__) && (defined(__SMALL__) || defined(__MEDIUM__))
+#if defined(__MEDIUM__))
```

Comment:
defined(__SMALL__) was removed. Is that related to one of the removed compilers? Is __MEDIUM__ unrelated? Seems like the whole if should go away since it's all predicated on the && __BORLANDC__.